.jenkins/pipelines/Azure/Linux/Jenkinsfile

```diff
@@ -174,12 +174,12 @@ def ACCHostVerificationTest(String version, String build_type) {
                            openssl ec -in keyec.pem -pubout -out publicec.pem
                            openssl genrsa -out keyrsa.pem 2048
                            openssl rsa -in keyrsa.pem -outform PEM -pubout -out publicrsa.pem
-                           ../../../output/bin/oecert --cert keyec.pem publicec.pem --out sgx_cert_ec.der --verify
-                           ../../../output/bin/oecert --cert keyrsa.pem publicrsa.pem --out sgx_cert_rsa.der --verify
-                           ../../../output/bin/oecert --report --out sgx_report.bin --verify
-                           ../../../output/bin/oecert --evidence --out sgx_evidence.bin --endorsements sgx_endorsements.bin --verify
-                           ../../../output/bin/oecert --evidence --verify --quote-proc in
-                           ../../../output/bin/oecert --evidence --verify --quote-proc out
+                           ../../../output/bin/oecert --format cert keyec.pem publicec.pem --out sgx_cert_ec.der --verify
+                           ../../../output/bin/oecert --format cert keyrsa.pem publicrsa.pem --out sgx_cert_rsa.der --verify
+                           ../../../output/bin/oecert --format legacy_report_remote sgx_report.bin --verify
```

Comment:
```suggestion
                           ../../../output/bin/oecert --format legacy_report_remote --out sgx_report.bin --verify
```

Reply:
Thanks! I may have deleted it by accident.